lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -31,6 +31,10 @@
 
     <p><input type="button" name="list-profiles-csv" value="${_("Download profile information as a CSV")}" data-endpoint="${ section_data['get_students_features_url'] }" data-csv="true"></p>
 
+    <p>${_("Click to generate a CSV file of all students who may enroll in this course but who have not yet signed up for it:")}</p>
```

Comment:
Change to:  "Click to generate a CSV file that lists learners who can enroll in the course but have not yet done so."